cmd/dockerd/daemon.go

```diff
@@ -260,6 +261,12 @@ func (cli *DaemonCli) start(opts daemonOptions) (err error) {
 		<-stopc // wait for daemonCli.start() to return
 	})
 
+	if b, err := json.Marshal(cli.Config); err != nil {
+		logrus.Errorf("Cannot formatting configuration in JSON: %s", err)
```

Comment:
I am afraid here `format` is better than `formatting`.
How about `logrus.Errorf("failed to format configuration in JSON: %v", err)` ?